src/internal/m365/collection/site/backup.go

```diff
@@ -151,16 +152,17 @@ func CollectLists(
 	logger.Ctx(ctx).Debug("Creating SharePoint List Collections")
 
 	var (
-		el   = errs.Local()
-		spcs = make([]data.BackupCollection, 0)
+		el        = errs.Local()
+		spcs      = make([]data.BackupCollection, 0)
+		selecting = []string{"id", "displayName"}
```

Comment:
If we're exposing this at a higher layer, we may want to also expose `idAnd`, so that this line looks like: 
```
    acc = api.CallConfig{Select: idAnd(displayName)}
```

```suggestion
		acc = api.CallConfig{Select: []string{"id", "displayName"}}
```